spec/lib/network/abstract_server_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'network/abstract_server'
+
+describe Network::AbstractServer do
+    context 'initialize' do
+        it 'returns true if AbstractServer inherits AbstractClient' do
+            expected = true
+
+            expect(expected).to eq Network::AbstractServer.ancestors.include?(Network::AbstractClient)
+        end
```

Comment:
I think this works if you are testing for true

```suggestion
            expect(Network::AbstractServer.ancestors.include?(Network::AbstractClient)).to be true
```

Reply:
also I believe the pattern for `Rspec` is:
```ruby
it ‘tests the thing’ do
  expected = 1

  actual = 1

  expect(actual).to eq(expected)
end
```

Reply:
It's looks more descriptive, I'll check when in my tests I can use this.